ts-sdk/whirlpool/tests/increaseLiquidity.test.ts

```diff
@@ -1,5 +1,480 @@
+      await setupTEPosition(whirlpools["Token-Token"], {
+        tickLower: 100,
+        tickUpper: 200,
+      }),
+    ];
+
+    positions["Token-TE Token"] = [
+      await setupPosition(whirlpools["Token-TE Token"]),
+      await setupPosition(whirlpools["Token-TE Token"], {
+        tickLower: 100,
+        tickUpper: 200,
+      }),
+      await setupTEPosition(whirlpools["Token-TE Token"]),
+      await setupTEPosition(whirlpools["Token-TE Token"], {
+        tickLower: 100,
+        tickUpper: 200,
+      }),
+    ];
+
+    positions["TE Token-TE Token"] = [
+      await setupPosition(whirlpools["TE Token-TE Token"]),
+      await setupPosition(whirlpools["TE Token-TE Token"], {
+        tickLower: 100,
+        tickUpper: 200,
+      }),
+      await setupTEPosition(whirlpools["TE Token-TE Token"]),
+      await setupTEPosition(whirlpools["TE Token-TE Token"], {
+        tickLower: 100,
+        tickUpper: 200,
+      }),
+    ];
+
+    positions["Token-TE Token with Transfer Fee extension"] = [
+      await setupPosition(
+        whirlpools["Token-TE Token with Transfer Fee extension"],
+      ),
+      await setupPosition(
+        whirlpools["Token-TE Token with Transfer Fee extension"],
+        {
+          tickLower: 100,
+          tickUpper: 200,
+        },
+      ),
+      await setupTEPosition(
+        whirlpools["Token-TE Token with Transfer Fee extension"],
+      ),
+      await setupTEPosition(
+        whirlpools["Token-TE Token with Transfer Fee extension"],
+        {
+          tickLower: 100,
+          tickUpper: 200,
+        },
+      ),
+    ];
+  });
+
+  const testLiquidityIncrease = async (
+    positionMint: Address,
+    tokenA: Address,
+    tokenB: Address,
+  ) => {
+    const amount = 10_000n;
+
+    const { quote, instructions } = await increaseLiquidityInstructions(
+      rpc,
+      positionMint,
+      { tokenA: amount },
+    );
+
+    const tokenBeforeA = await fetchToken(rpc, ataMap[tokenA]);
+    const tokenBeforeB = await fetchToken(rpc, ataMap[tokenB]);
+    await sendTransaction(instructions);
+    const positionAddress = await getPositionAddress(positionMint);
+    const position = await fetchPosition(rpc, positionAddress[0]);
+    const tokenAfterA = await fetchToken(rpc, ataMap[tokenA]);
+    const tokenAfterB = await fetchToken(rpc, ataMap[tokenB]);
+    const balanceChangeTokenA =
+      tokenBeforeA.data.amount - tokenAfterA.data.amount;
+    const balanceChangeTokenB =
+      tokenBeforeB.data.amount - tokenAfterB.data.amount;
+
+    assert.strictEqual(quote.tokenEstA, balanceChangeTokenA);
+    assert.strictEqual(quote.tokenEstB, balanceChangeTokenB);
+    assert.strictEqual(quote.liquidityDelta, position.data.liquidity);
+  };
+
+  it("Should handle liquidity increase for whirlpool with tokenA=Token and tokenB=Token, and Centered Position", async () => {
+    await testLiquidityIncrease(
+      positions["Token-Token"][0],
+      address(mintA),
```

Comment:
nit: Aren't these already and address? Why do you need to convert to address again?